UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -251,6 +260,15 @@ public void Update()
 		}
 
 
+		public static bool IsWithinBounds(BoundsInt boundsInt, Vector3 Point)
+		{
+			if (Point.x > boundsInt.xMin && Point.x < boundsInt.xMax  &&
+			    Point.y > boundsInt.yMin && Point.y < boundsInt.yMax)
+				return true;
+
+			return false;
```

Comment:
there's a BoundsInt.Contains(Vector3Int position) you can use instead of creating this